app/types/clusterconfig/clusterconfig.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package status;
+option go_package  = "../clusterconfig";
+
+message ClusterConfig {
```

Comment:
one thing that we will want in the future is a way to fetch KSM version and fetch TLS certificate information. 

that's my mistake for not mentioning those things. but, we can add those in a follow up PR